hpack/definitions.yaml

```diff
@@ -70,6 +70,7 @@ _definitions:
         - -Wno-missing-local-signatures
         - -Wno-monomorphism-restriction
         - -Wno-implicit-prelude
+        - -Wno-unused-do-bind
```

Comment:
1. I am not sure if it is really necessary in Indigo code. It was quite needed in case of Lorentz, need to check that for Indigo.
2. In any case I think it's better to do it on per-target basis. Even if it makes sense for Indigo code, it seems to make little sense in executables and tests (there it should at least be on per-module basis if appears to be needed somewhere).